### DISCUSSION TITLE
Apprise notification broken ?

### DISCUSSION THREAD

**Discussion by croisez** [mcguirepr89/BirdNET-Pi#527] [Q&A] [answered]
Hi BirdNET-Pi users,
I have a PI3 with BirdNET-Pi up-to-date.
I have configured Apprise notification system for Telegram, and setup my own bot in Telegram to be warned of new notifications in my channel.
The Test notification is working well, and the notif comes well at my smartphone.

```
2022-09-01 09:25:35,983 - INFO - Notifying 1 service(s) asynchronously.
2022-09-01 09:25:36,357 - INFO - Sent Telegram notification.
2022-09-01 09:25:37,939 - INFO - Sent Telegram attachment: https://live.staticflickr.com/7430/27545810581_8bfa8289a3_c.jpg?rto=4.0&cto=4.0&verify=yes&cache=yes.
```

But, no notification is coming to my Telegram app when a bird detection occurs.
Is there another configuration which I should do, or a service that I should start ?

**Comment by matthew73210** [comment, marked as answer]
Hey,

When you go back to settings, what have you ticked in 'Notifications' ?

**Reply by croisez** [reply]
You are very right, I did not select any tick.
I thought that there was something pre-selected by default.
This is not the case. Now that I have selected one of the ticks, I have received notifications thru Apprise API.
It works.
